go.mod

```diff
@@ -36,6 +36,7 @@ require (
 require (
 	github.com/go-co-op/gocron v1.37.0
 	github.com/kyma-project/template-operator/api v0.0.0-20240404131948-52c84f14e73c
+	github.com/mandelsoft/goutils v0.0.0-20241005173814-114fa825bbdc
```

Comment:
please be careful what is imported, this is not what we want.

Reply:
It shouldn t be added this line!